packages/web-components/src/progress-bar/progress-bar.options.ts

```diff
@@ -0,0 +1,47 @@
+import { ValuesOf } from '@microsoft/fast-foundation';
+
+/**
+ * ProgressBarThickness Constants
+ * @public
+ */
+export const ProgressBarThickness = {
+  medium: 'medium',
+  large: 'large',
+} as const;
+
+/**
+ * Applies bar thickness to the content
+ * @public
+ */
+export type ProgressBarThickness = ValuesOf<typeof ProgressBarThickness>;
+
+/**
+ * ProgressBarShape Constants
+ * @public
+ */
+export const ProgressBarShape = {
+  rounded: 'rounded',
+  rectangular: 'rectangular',
```

Comment:
this is interesting - @miroslavstastny curious if this is intentionally different than `square` semantic found in badge, etc. Are we documenting these inconsistencies elsewhere or are some of these known and desired to be "aligned" back to something consistent?

Reply:
not blocking but if we have an opportunity to do that work now...

Reply:
My first guess would be "square" means "four equal straight sides and four right angles", which might be the case for `Badge` or `Avatar` where we use it in FUIR9, but we also use it in `Button` and `Image` which can render as rectangles.

`ProgressBar` is still unstable in FUIR9, I will discuss with engineers there.

Reply:
FUIR9 will rename to `square`

Reply:
Will rename in this implementation.